src/transformers/models/x_clip/modeling_x_clip.py

```diff
@@ -1061,21 +1061,46 @@ def forward(
         Examples:
 
         ```python
-        >>> from PIL import Image
-        >>> import requests
-        >>> from transformers import XCLIPProcessor, XCLIPVisionModel
+        >>> from decord import VideoReader, cpu
```

Comment:
I'm not sure this depenedency is installed on the machine running the doc tests, so it probably needs to be added.

Reply:
Yes it does, VideoMAE is already using it (cc @ydshieh)